api/src/public/login/script.js

```diff
@@ -1,40 +1,35 @@
+/* eslint-disable */
+import {
+  setState,
+  request,
+  getCookie,
+  getLocale,
+  parseTranslations,
+  baseTranslate,
+  togglePassword,
+  getUserCtx
+} from './auth-utils.js';
+
```

Comment:
I'm nervous about all the modern code you've added here because we don't have any transpilation to ensure backwards compatibility. Have you checked that all of these features are supported?

Reply:
I've checked for Chrome v.90 and Firefox v.74

Arrow functions [yes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#browser_compatibility)
Optional chaining [yes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#browser_compatibility)
Export [yes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export#browser_compatibility)
Import [yes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#browser_compatibility)

But the one for imports shows a higher support version for `Available in service workers` which is v.91 for Chrome.